### PR TITLE
[#960] fix(dashboard): simplify dependency and correct the startup script

### PR DESCRIPTION
**Pull request by yl09099**
### What changes were proposed in this pull request?

simplify dependency and correct the startup script.

### Why are the changes needed?

(Please clarify why the changes are needed. For instance,

Fix: #960 

### Does this PR introduce _any_ user-facing change?

(Please list the user-facing changes introduced by your change, including

No.

### How was this patch tested?

Needn't.

### CODE REVIEW

**Comment by yl09099** [comment]
> What's this pr the relationship of #1326

Same PR. I'm changing it after the merger.

**Comment by yl09099** [comment]
> #1326

Some fixes based on PR #1326.I'm making changes after the merger.

**Comment by yl09099** [comment]
@zuston @xianjingfeng @jerqi  Modify the dashboard pom dependencies, modify the startup script to be independent of Hadoop, and separate the dashboard configuration file from the Coordinator configuration file so that it can be deployed independently.

**Review by zuston — DISMISSED** [review]
LGTM.

**Comment by xianjingfeng** [comment]
Could you fix these two problems in the pr? @yl09099 
https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825

**Comment by yl09099** [comment]
> Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)

This has been modified.

**Comment by xianjingfeng** [comment]
> > Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)
> 
> This has been modified.

It seems that one of them was be fixed. But we still must set the env `HADOOP_HOME`, right?

**Comment by yl09099** [comment]
> > > Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)
> > 
> > 
> > This has been modified.
> 
> It seems that one of them was fixed. But we still must set the env `HADOOP_HOME`, right?

I have deleted the configurations related to start-dashboard and HDFS, and do not need to set HADOOP_HOME on the dashboard.

**Comment by yl09099** [comment]
> > > Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)
> > 
> > 
> > This has been modified.
> 
> It seems that one of them was fixed. But we still must set the env `HADOOP_HOME`, right?

I have deleted the HDFS configuration in start-dashboard and do not need to set HADOOP_HOME on dashboard.

**Comment by xianjingfeng** [comment]
> > > > Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)
> > > 
> > > 
> > > This has been modified.
> > 
> > 
> > It seems that one of them was fixed. But we still must set the env `HADOOP_HOME`, right?
> 
> I have deleted the HDFS configuration in start-dashboard and do not need to set HADOOP_HOME on dashboard.


Is there anything problems with the following code?
https://github.com/apache/incubator-uniffle/blob/ecbf2e79423505dd44bf54c612572c5c9da71175/bin/utils.sh#L175-L178

**Comment by yl09099** [comment]
> > > > > Could you fix these two problems in this pr? @yl09099 [#1339 (comment)](https://github.com/apache/incubator-uniffle/pull/1339#issuecomment-1831538825)
> > > > 
> > > > 
> > > > This has been modified.
> > > 
> > > 
> > > It seems that one of them was fixed. But we still must set the env `HADOOP_HOME`, right?
> > 
> > 
> > I have deleted the HDFS configuration in start-dashboard and do not need to set HADOOP_HOME on dashboard.
> 
> Is there anything problems with the following code?
> 
> https://github.com/apache/incubator-uniffle/blob/ecbf2e79423505dd44bf54c612572c5c9da71175/bin/utils.sh#L175-L178

Learned, changed

File: bin/utils.sh [hunk 1]
```diff
@@ -172,7 +177,9 @@ function load_rss_env {
     echo "No env JAVA_HOME."
     exit 1
   fi
-  if [ -z "$HADOOP_HOME" ]; then
+  if [[ $is_dashboard -eq 0 || -z "$HADOOP_HOME" ]]; then
+    echo "Dashboard need not HADOOP_HOME."
+  elif [[ -z "$HADOOP_HOME" ]]; then
     echo "No env HADOOP_HOME."
     exit 1
   fi
```
Comment (xianjingfeng) [inline bin/utils.sh:185]:
```suggestion
  if [[ -z "$HADOOP_HOME" ]]; then
    if [[ $is_dashboard -eq 0 ]]; then
      echo "Dashboard need not HADOOP_HOME."
    else
      echo "No env HADOOP_HOME."
      exit 1
    fi
  fi
```

File: bin/utils.sh [hunk 2]
```diff
@@ -152,6 +152,11 @@ function is_port_in_use {
 function load_rss_env {
   set -o allexport
 
+  is_dashboard=1
+  if [ $# -eq 1 ]; then
+          is_dashboard=$1
+  fi
```
Comment (xianjingfeng) [inline bin/utils.sh:158]:
```suggestion
  is_dashboard=0
  if [ $# -eq 1 ]; then
    is_dashboard=1
  fi
```

Comment (xianjingfeng) [reply bin/utils.sh:158]: @zuston @jerqi Do you have any idea about this part?

**Review by xianjingfeng — APPROVED** [review]
LGTM

**Comment by zuston** [comment]
Thanks @yl09099 @xianjingfeng

**Pull request merged**